### PR TITLE
Margin fixes

### PR DESCRIPTION
**Pull request by ismarS**
As per mentioned here https://github.com/yahoo/pure-site/issues/305 fixed the margin on the right. 

While looking into it I've found another minor issue that needed fixing, the `ul` in `blog.css` having padding on the left (more noticeable in phone/tablet view).

Should work fine now. Happy to make more changes if I missed something.

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: public/css/layouts/gallery.css [hunk 1]
```diff
@@ -123,4 +123,8 @@ h1,h2,h3,h4,h5,h6 {
     .photo-box-thin {
         height: 250px;
     }
+
+    .l-box {
+    padding: 2em;
```
Comment (redonkulus) [inline public/css/layouts/gallery.css]: can you fix the spacing here? should be indented another 4 spaces

File: public/css/layouts/gallery.css [hunk 2]
```diff
@@ -123,4 +123,8 @@ h1,h2,h3,h4,h5,h6 {
...
+}
```
Comment (redonkulus) [inline public/css/layouts/gallery.css]: indent this 4 spaces

File: public/css/layouts/pricing.css [hunk 3]
```diff
@@ -36,6 +36,10 @@ p {
     box-shadow: 0 1px 1px rgba(0,0,0, 0.10);
 }
 
+.pure-menu-link, .pure-menu-disabled, .pure-menu-heading {
```
Comment (redonkulus) [inline public/css/layouts/pricing.css]: can you list each selector on another line?

File: public/css/layouts/gallery.css (hunk 1 above)
Comment (ismarS) [inline public/css/layouts/gallery.css]: Will do.

**Comment by redonkulus** [comment]
@ismarS any update on this?

**Comment by ismarS** [comment]
@redonkulus got caught up elsewhere. Wouldn't mind if someone else wrapped it up. Otherwise, I'll get on it when I find the time.

**Comment by redonkulus** [comment]
No problem. I can take a look at finishing this up. Thanks!

**Comment by redonkulus** [comment]
Opened a new PR to fix the issues https://github.com/yahoo/pure-site/pull/349

**Pull request closed without merging**